_data/software/browsers-desktop/1_firefox.yml

```
@@ -17,7 +17,7 @@ description: |
   </ul>
   You can still stay logged into websites by allowing exceptions.
 
-  ##### **Disable Search Suggestions**
+  ##### **Disable Search Suggestions (U.S. Only)**
```

It would be better to put just say that these features may not be available depending on one's region, just in case the feature expands to more areas in the future.
```suggestion
  ##### **Disable Search Suggestions**
   *These features may not be available depending on your region.*
```

---

Yes, as my Firefox do contain Search Suggestions section but doesn't have the options named like you have mentioned. My firefox says:

**Search Suggestions**
Choose how suggestions from search engines appear.
```
Provide search suggestions
Show search suggestions in address bar results
Show search suggestions ahead of browsing history in address bar results
Show search suggestions in Private Windows
```